docs/source/models/concept_models/vivarium_mncnh_portfolio/concept_model.rst

```diff
@@ -1798,6 +1851,30 @@ Default stratifications to all observers should include scenario and input draw.
       * Use the interactive sim to confirm there is no coverage of ACS outside of the eligible gestational age range.
     - 
     - 
+  * - 17.0
+    - * Confirm that baseline coverage of IFA matches the health systems team estimates of IFA at ANC as specified in the :ref:`oral iron supplementation intervention page <oral_iron_antenatal>`.
+      * Confirm that baseline coverage of MMS is 0 in all locations.
+      * Confirm that only simulants who attend ANC receive IFA in all scenarios.
+      * Use the interactive sim to confirm that in the baseline scenario, the exposure distribution of hemoglobin and anemia among PLW and WRA as well as the morbidity and mortality rates of each maternal disorder matches that of GBD.
+      * Use the interactive sim to confirm that maternal hemoglobin exposures stratified by IFA coverage match supplementation effect sizes.
+      * Use the interactive sim to confirm that in the baseline scenario, the exposure distribution of LBWSG matches that of GBD.
+      * Confirm that in the baseline scenario, the mortality rates of each neonatal disorder among both neonatal age groups continue to match that of GBD.
+      * Confirm that the rate of each birth outcome in the baseline scenario continues to validate to GBD estimates.
+      * Confirm that the birth outcome rates stratified by coverage of each supplementation regimen matches the expected supplementation effect sizes.
+      * Use the interactive sim to confirm that GA and BW differences between regimens match the effect sizes of each supplementation regimen (i.e., none, only IFA, both IFA and MMS).
+    - 
+    -
+  * - 18.0
+    - * Confirm that coverage of IV iron among the eligible population verifies to the scenario-specific level as specified in the :ref:`IV iron intervention page <intervention_iv_iron_antenatal_mncnh>`.
+      * Confirm that coverage of IV iron is 0 among the non-eligible population.
```

Comment:
How does one check this? I think IV iron eligibility is based on hemoglobin, but that is modified by IV iron.